.pre-commit-config.yaml

```
@@ -18,25 +18,25 @@ repos:
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
 -   repo: https://github.com/pycqa/pylint
-    rev: pylint-2.7.1
+    rev: v2.11.1
     hooks:
     -   id: pylint
         name: pylint (library code)
         types: [python]
-        exclude: "^(docs/|tests/|examples/|setup.py$)"
--   repo: local
-    hooks:
-    -   id: pylint_examples
-        name: pylint (examples code)
+        args:
+          - --disable=consider-using-f-string
```

If the PR fails on duplicate code, please update this line to the following.
```suggestion
          - --disable=consider-using-f-string,duplicate-code
```

---

Oh I missed that, good catch!